pkg/storage/backend-storage/backend-storage.go

```diff
@@ -98,14 +98,25 @@ func (bs *BackendStorage) getStorageClass() (string, error) {
 		return storageClass, nil
 	}
 
+	k8sDefault := ""
+	kvDefault := ""
 	for _, obj := range bs.scStore.List() {
 		sc := obj.(*storagev1.StorageClass)
+		if sc.Annotations["storageclass.cdi.kubevirt.io/is-default-class"] == "true" {
+			kvDefault = sc.Name
+		}
 		if sc.Annotations["storageclass.kubernetes.io/is-default-class"] == "true" {
-			return sc.Name, nil
+			k8sDefault = sc.Name
 		}
 	}
 
-	return "", fmt.Errorf("no default storage class found")
+	if kvDefault != "" {
+		return kvDefault, nil
+	} else if k8sDefault != "" {
+		return k8sDefault, nil
+	} else {
+		return "", fmt.Errorf("no default storage class found")
+	}
```

Comment:
Not a fan of if / else if / else if :) 

IDK just an alternative as proposal, don't really know if this is better:
```
	k8sDefault := ""
	for _, obj := range bs.scStore.List() {
		sc := obj.(*storagev1.StorageClass)
		if sc.Annotations["storageclass.cdi.kubevirt.io/is-default-class"] == "true" {
			return sc.Name, nil
		}
		if sc.Annotations["storageclass.kubernetes.io/is-default-class"] == "true" {
			k8sDefault = sc.Name
		}
	}

	if k8sDefault != "" {
		return k8sDefault, nil
	}
	return "", fmt.Errorf("no default storage class found")
```

Reply:
Feel free to choose the one you prefer

Reply:
My first version actually looked exactly like the above :)
I went with the if / else if / else approach instead to highlight what the algorithm does: "pick the virt class if it exists, else pick the k8s one if it exists, else fail".
It also feels too trivial for a unit test...
Please hold/unlgtm if you feel strongly about either.
Thanks for the review!

Reply:
That's fine as is :)